packages/ui/src/utils/dom/getAppliedStyles.ts

```diff
@@ -0,0 +1,16 @@
+import { kebabCase } from 'lodash';
+
+export default function getAppliedStyles(
```

Comment:
Does this belong in `ui` package? If this is only used by square then it should be in that package.

Reply:
Do you mean, to move it to the Square package? I think it could be used by other packages in the future.

https://github.com/bigcommerce/checkout-js/blob/cdc374a082af0ae5a53c9de6255571eb3b937719/packages/core/src/app/payment/paymentMethod/StripeUPEPaymentMethod.tsx#L70